src/python/gem5/components/boards/abstract_board.py

```diff
@@ -141,6 +153,15 @@ def get_memory(self) -> "AbstractMemory":
         """
         return self.memory
 
+    def get_mem_ranges(self) -> Sequence[AddrRange]:
+        """Get all the mem ranges in the board, This
+        tries to account for boards shipping memories other
```

Comment:
```suggestion
        tries to account for boards skipping memories other
```
I think this is what you meant ;)

Reply:
Nono I meant shipping, but it's probably bad translation from italian. Will reword it